src/tests/Interop/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ endif(CLR_CMAKE_TARGET_WIN32)
 if(CLR_CMAKE_TARGET_UNIX)
     add_subdirectory(COM/ComWrappers/MockReferenceTrackerRuntime)
     add_subdirectory(COM/ComWrappers/WeakReference)
+    add_subdirectory(GCBridge)
 endif(CLR_CMAKE_TARGET_UNIX)
```

Comment:
Remove the `if(CLR_CMAKE_TARGET_UNIX)` itself. The three `add_subdirectory()` can be moved to line 66. Remove the two lines at 86 and 86 that are only for Windows. All three build everywhere.